<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="sentMessageBox">
        <property name="title">
         <string>Отправленные</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QGroupBox" name="getMessageBox">
           <property name="title">
            <string>Полученные</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="2" column="1">
             <widget class="QPlainTextEdit" name="getMessagePlain"/>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="getFramesCheckBox">
              <property name="text">
               <string>Get frames</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string/>
              </property>
              <widget class="QLabel" name="lengthLabel">
               <property name="geometry">
                <rect>
                 <x>64</x>
                 <y>20</y>
                 <width>67</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>L</string>
               </property>
              </widget>
              <widget class="QLabel" name="dataLabel">
               <property name="geometry">
                <rect>
                 <x>140</x>
                 <y>20</y>
                 <width>67</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Data</string>
               </property>
              </widget>
              <widget class="QLabel" name="idLabel">
               <property name="geometry">
                <rect>
                 <x>27</x>
                 <y>20</y>
                 <width>67</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>ID</string>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPlainTextEdit" name="sentMessagePlain"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="id">
               <property name="text">
                <string>ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="idLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="placeholderText">
                <string>123</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="data">
               <property name="text">
                <string>Data</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="dataLineEdit">
               <property name="placeholderText">
                <string>AA BB CC DD</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="sendButon">
               <property name="text">
                <string>Send</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>633</width>
     <height>22</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDisconnect">
    <property name="title">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuClear">
    <property name="title">
     <string>Clear</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuConnect"/>
   <addaction name="menuDisconnect"/>
   <addaction name="menuClear"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionClear"/>
   <addaction name="actionInformation"/>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/images/usbcable_78354.png (disconnect)</normaloff>:/actions/images/usbcable_78354.png (disconnect)</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/images/closingsession_theoutputnear_cierredesesion_10028.png (exit)</normaloff>:/actions/images/closingsession_theoutputnear_cierredesesion_10028.png (exit)</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/images/actions_edit_clear_15784.png (clear)</normaloff>:/actions/images/actions_edit_clear_15784.png (clear)</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/images/Settings_30027.png (settings)</normaloff>:/actions/images/Settings_30027.png (settings)</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/actions/images/info.png (info)</normaloff>:/actions/images/info.png (info)</iconset>
   </property>
   <property name="text">
    <string>Information</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
